packages/repository/test/unit/decorator/model-and-relation.decorator.unit.ts

```diff
@@ -347,6 +374,17 @@ describe('model decorator', () => {
           property.ERR_PROP_NOT_ARRAY,
         );
       });
+
+      it('throws when properties are cyclic', () => {
```

Comment:
According to the tests here, does it mean people couldn't import 2 related models in one controller file?

Reply:
Actually, the person wouldn't be able to import either of the models at all since they are both related

Reply:
Had a talk with @shimks , and thanks for explaining what this test case aims to catch.

Here is a summary:

This test is to assert that a proper error is thrown when users define the relation in a wrong way, like:
```ts
@model()
Customer{
  // The correct signature is using a type resolver 
  // @hasMany(()=>Order)
  @hasMany(Order)
  orders
}
```